### ISSUE TITLE
[PATCH] Fix NAME section

### ISSUE THREAD

**Issue by utkarsh2102** [shlomif/perl-file-find-object-rule#1]
In Debian, we're using the following patch, thought you might be interested in the same?

```
--- a/lib/File/Find/Object/Rule.pm
+++ b/lib/File/Find/Object/Rule.pm
@@ -544,6 +544,10 @@
 
 =encoding UTF-8
 
+=head1 NAME
+
+File::Find::Object::Rule - Alternative interface to File::Find::Object
+
 =head1 VERSION
 
 version 0.0311
@@ -576,10 +580,6 @@
 considered Alpha software. Please report any problems with
 L<File::Find::Object::Rule> to its RT CPAN Queue.
 
-=head1 NAME
-
-File::Find::Object::Rule - Alternative interface to File::Find::Object
-
 =head1 METHODS
 
 =over
--- a/lib/File/Find/Object/Rule/Extending.pod
+++ b/lib/File/Find/Object/Rule/Extending.pod
@@ -6,6 +6,10 @@
 
 =encoding UTF-8
 
+=head1 NAME
+
+File::Find::Object::Rule::Extending - the mini-guide to extending File::Find::Object::Rule
+
 =head1 VERSION
 
 version 0.0311
@@ -76,10 +80,6 @@
 consumed C<in> as a parameter to it, then got all confused as it
 doesn't know about a C<'.'> rule.
 
-=head1 NAME
-
-File::Find::Object::Rule::Extending - the mini-guide to extending File::Find::Object::Rule
-
 =head1 NOTES ABOUT THE CALLBACK
 
 The callback can access the L<File::Find::Object::Result> using
--- a/lib/File/Find/Object/Rule/Procedural.pod
+++ b/lib/File/Find/Object/Rule/Procedural.pod
@@ -6,6 +6,10 @@
 
 =encoding UTF-8
 
+=head1 NAME
+
+File::Find::Object::Rule::Procedural - File::Find::Object::Rule's procedural interface
+
 =head1 VERSION
 
 version 0.0311
@@ -61,10 +65,6 @@
                    size    => '>20M',
                    in      => $ENV{HOME} );
 
-=head1 NAME
-
-File::Find::Object::Rule::Procedural - File::Find::Object::Rule's procedural interface
-
 =head1 AUTHOR
 
 Richard Clamp <richardc@unixbeard.net>
```

**Comment by utkarsh2102** [comment]
cf: https://salsa.debian.org/perl-team/modules/packages/libfile-find-object-rule-perl/blob/master/debian/patches/pod-whatis.patch

**Comment by shlomif** [comment]
@utkarsh2102 : hi, this seems to be a quirk of https://metacpan.org/pod/Pod::Weaver::PluginBundle::SHLOMIF . I'll try to ask how to fix it.

**Comment by gregoa** [comment]
This seems to be fixed in 0.0312 with 99dafed41960331de3c39905a16f95c52233dfde -  thank you!
I'm seeing no glitches anymore after removing our patch, so I guess the ticket can be closed.

**Comment by shlomif** [comment]
> This seems to be fixed in 0.0312 with [99dafed](https://github.com/shlomif/perl-file-find-object-rule/commit/99dafed41960331de3c39905a16f95c52233dfde) - thank you!
> I'm seeing no glitches anymore after removing our patch, so I guess the ticket can be closed.

Closing, thanks!
